rootage.html

```diff
@@ -0,0 +1,118 @@
+              // ported by UngluedBike
+              name: 'Premium Free Timer Freeze',
+              patches: [{offset: 0x16E62D, off: [0xFF, 0xC8], on: [0x90, 0x90]}]
+            },
+            {
+              // ported by UngluedBike
+              name: 'Standard/Menu Timer Freeze',
+              patches: [{offset: 0x384097, off: [0x74], on: [0xEB]}]
+            },
+            {
+              // ported by UngluedBike
+              name: '90sec Music Select Timer',
+              tooltip : "Make sure your Select Time option is set to \"45 SEC\" in the Game Options for this to work!",
+              patches: [{offset: 0xFD204, off: [0x2D], on: [0x5A]}]
+            },
+            {
+              // ported by UngluedBike
+              name: 'Cursor lock',
+              tooltip : "After song finishes, song select remains on previous song",
+              patches: [{offset: 0x332314, off: [0x74, 0x1D], on: [0x90, 0x90]}]
+            },
+            {
+              // ported by UngluedBike
+              name: 'CS-Style Song Start Delay',
+              tooltip : "Holding Start will pause the song at the beginning until you release it",
+              patches: [{offset: 0x35021A, off: [0x7D, 0x25], on: [0x90, 0x90]}]
+            },
+            {
+              // ported by dogelition_man
+              name: 'Play video preview on all songs',
+              tooltip : "By default, only some songs in the beginner folder show a BGA preview on the music select screen. This edit extends this function to every song in any folder.",
+              patches: [
+                {offset: 0x11F21E, off: [0x74, 0x0E], on: [0x90, 0x90]},
+                {offset: 0x11BE6C, off: [0x0F, 0xB6], on: [0xEB, 0x2F]}
+              ]
+            },
+            {
+			  // ported by UngluedBike
+              name: 'Hide INSERT COIN[S] text',
+              patches: [{offset : 0x106859, off: [0x9B], on: [0x9A]}]
+            },
+            {
+			  // ported by UngluedBike
+              name: 'Hide CREDIT: %d text',
+              patches: [{offset: 0x1063E7, off: [0x5D], on: [0x5C]}]
+            },
+            {
+			  // ported by UngluedBike
+              name: 'Hide CREDIT: %d COIN %d / %d text',
+              patches: [{offset: 0x1063BD, off: [0x67], on: [0x66]}],
+            },
+            {
+			  // ported by UngluedBike
+              name: 'Hide EXTRA PASELI: %d text',
+              patches: [{offset: 0x1066A8, off: [0xF4], on: [0xF3]}]
+            },
+            {
+			  // ported by UngluedBike
+              name: 'Hide PASELI: NO ACCOUNT text',
+              patches: [{offset: 0x106762, off: [0x7A], on: [0x79]}]
+            },
+            {
+			  // ported by UngluedBike
+              name: 'Hide PASELI: ****** text',
+              patches: [{offset: 0x1066CB, off: [0xF1], on: [0xF0]}]
+            },
+          ], "2019-04-18"),
+        ]);
+      });
+    </script>
+</head>
+<body>
+<h1>IIDX Cannon Ballers DLL Modder</h1>
```

Comment:
🤔

Reply:
Ah, my mistake. Thanks for pointing it out.

Reply:
I've made the appropriate changes now.